proposals/incubation/strimzi.md

```diff
@@ -0,0 +1,123 @@
+# Strimzi Incubation Stage Proposal
+
+## Introduction
+
+Strimzi is a project focused on running [Apache Kafka][ak] on [Kubernetes][kube].
+Apache Kafka is the _de facto_ standard platform for building real-time data pipelines.
+It provides support for high-throughput / low-latency messaging, as well as sophisticated development options that cover all the stages of a distributed data streaming pipeline, from ingestion, through storage to processing.
+But running it on Kubernetes can be complex and tedious.
+Strimzi makes it easy by using the operator pattern and declarative deployments.
+It provides operators not just for deploying Kafka clusters, but also for managing topics, users and connectors to make sure running Apache Kafka feels Kube-native.
+
+Strimzi also integrates with many other CNCF projects such as:
+
+* Kubernetes
+* [Prometheus][prom] for metrics and monitoring
+* [Open Policy Agent][opa] for authorization
+* [KEDA][keda] for autoscaling of Kafka clients
+* [Jaeger][jaeger] and [OpenTelemetry][otel] for tracing
+
+In addition, Strimzi is used by [Knative][knative] to provide its Kafka source.
+
+Strimzi can be installed using [Helm][helm] charts or using [OperatorHub.io][operatorhub] which is part of the [Operator Framework][operatorframework] project.
+
+Strimzi joined CNCF Sandbox in September 2019
+
+## Statement on alignment with the CNCF mission
+
+> The Foundation's mission is to make cloud native computing ubiquitous
+
+While Kafka is _capable_ of horizontal scalability, achieving such things as zero-downtime upgrades, and scaling clusters within Kubernetes without an operator is too high a barrier for many organizations.
+This prevents ubiquitous use of Kafka anywhere within the cloud.
```

Comment:
_within the cloud_ sounds a bit weird to me. What about something like _with cloud native platforms such as Kubernetes_